shared_model/validators/transactions_collection/transactions_collection_validator.cpp

```diff
@@ -57,6 +59,17 @@ namespace shared_model {
         }
       }
 
+      interface::TransactionBatchParserImpl batch_parser;
+      BatchValidator batch_validator;
+
+      auto batches = batch_parser.parseBatches(transactions);
+      for (auto &batch : batches) {
+        if (auto answer = batch_validator.validate(batch)) {
+          reason.second.emplace_back(answer.reason());
+          break;
```

Comment:
If several batches are invalid, there will be an error only for one of them. Consider removing break.

Reply:
Thanks!